camel/agents/chat_agent.py

```diff
@@ -200,8 +211,7 @@ def is_function_calling_enabled(self) -> bool:
         """
         return len(self.func_dict) > 0
 
-    def update_memory(self, message: BaseMessage,
-                      role: OpenAIBackendRole) -> None:
+    def update_memory(self, message: BaseMessage, role: OpenAIBackendRole) -> None:
         r"""Updates the agent memory with a new message.
```

Comment:
The length of this line is over 80, which does not conform to Python's programming standards. So please make sure you run the command `pre-commit install` before submitting your code.